wordpress-mobile/WordPress-iOS
# Target 4.28.1 branch to fix Popular Plugins from loading

Fixes #16056
WPKit Related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/370

| Popular Plugins | Searching |
|:---:|:---:|
|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-11 at 14 49 34](https://user-images.githubusercontent.com/793774/110846178-7a555b00-8279-11eb-9f55-34b71f37b142.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-11 at 14 49 48](https://user-images.githubusercontent.com/793774/110846188-7c1f1e80-8279-11eb-8ecc-53691da5458c.png)|

### To test:
1. Launch the app
2. Tap on My Sites
3. Tap on a Jetpack site
4. Tap on Plugins
5. Swipe down to verify the popular plugins appear
6. Tap See All 
7. Verify the plugins load
8. Tap back
9. Swipe up to show search
10. Search for a plugin 
11. Verify the results appear

PR submission checklist:
